### PR TITLE
Modify time format for flexformat recipe interpreter

### PR DESCRIPTION
**Pull request by rwdavis513**
The variable type for start_time and now_time did not match the simple recipe interpreter, which is now fixed. 

Also, added more tests to verify functionality.

### CODE REVIEW

File: nodes/recipe_handler.py [hunk 1]
```diff
@@ -246,6 +246,7 @@ def save_recipe_dp(self, variable):
 #------------------------------------------------------------------------------
 # Our ROS node main entry point.  Starts up the node and then waits forever.
 if __name__ == '__main__':
+    pub_debug = rospy.Publisher('debug/recipe_handler', String, queue_size=10)
```
Comment (rbaynes) [inline nodes/recipe_handler.py]: Do we want to make `pub_debug` conditional on `TRACE`?

File: src/openag_brain/recipe_interpreters.py [hunk 2]
```diff
@@ -58,20 +68,39 @@ def interpret_simple_recipe(recipe, start_time, now_time):
...
-    Recipe Interpreter should read a recipe, current_time, start_time, variable and return a value.
+    Recipe Interpreter should read a recipe, now_time, start_time, variable and return a value.
        Determine the time since the beginning of the recipe.
        Determine what is the current step
        Calculate the remaining time left in this step.
        Look up the value within that step for that variable.
     """
-
+    _id = recipe["_id"]
+    rospy.loginfo(recipe["phases"])
```
Comment (rbaynes) [inline src/openag_brain/recipe_interpreters.py]: should be logdebug() ?

File: src/openag_brain/recipe_interpreters.py [hunk 3]
```diff
@@ -83,6 +112,28 @@ def interpret_flexformat_recipe(recipe, current_time, start_time):
     )
 
 
+def verify_time_units(time_var):
```
Comment (rbaynes) [inline src/openag_brain/recipe_interpreters.py]: 👍

File: src/openag_brain/utils.py [hunk 4]
```diff
@@ -4,6 +4,7 @@
 from random import randint
 from re import match
 import json
+import rospy
```
Comment (rbaynes) [inline src/openag_brain/utils.py]: Do we need this import?

File: src/openag_brain/utils.py [hunk 5]
```diff
@@ -45,7 +46,7 @@ def read_environment_from_ns(namespace):
 # Turn on logic tracing by making the variable below True.
 # Output ONLY is written to this node's log file:
 # tail -f ~/.ros/log/latest/environments-environment_1-recipe_handler_1-6.log
-TRACE = False   # TODO: Move to settings file?
+TRACE = True   # TODO: Move to settings file?
```
Comment (rbaynes) [inline src/openag_brain/utils.py]: Should we check this in as False?

**Comment by rwdavis513** [comment]
@rbaynes Thanks for catching those items. I've updated the code and pushed my latest commits. Could you take another look?

**Pull request merged**